tutorials/javaFxPart3.md

```diff
@@ -15,38 +15,40 @@ JavaFX has an _event-driven architecture style_. As such, we programmatically de
 
 For Duke, there are two events that we want to respond to, namely the user pressing `Enter` in the `TextField` and left-clicking the `Button`. These are the `onAction` event for the `TextField` and the `onMouseClicked` event for the `Button`.
 
-For now, let’s have the application add a new `Label` with the text from the `TextField`. Update the `Main` class as follows. You'll need to add an `import javafx.scene.control.Label;` too.
+For now, let’s have the application add a new `DialogBox` with the text from the `TextField`, so the user can see the message they have sent. Update the `Duke` class as follows.
 ```java
 @Override
 public void start(Stage stage) {
-    // Step 1 code here
+    //Step 1 code here
 
     //Step 2 code here
+    //Delete or comment off the following section
+    //Label text = new Label("Hello!");
+    //ImageView displayPicture = new ImageView(user);
+    //DialogBox dialogBox = new DialogBox(text, displayPicture);
```

Comment:
Correct me if im wrong, but I can't seem to find the `DialogBox` class in part 1 or 2.
Would it be wise to introduce this step 2 block of code here first when `DialogBox` class is only introduced later on in iteration 2?

Reply:
I believe @laney0808 is intending to shift the `DialogBox` class to part 2 so that the readers can learn about Vbox, Image View and Labels as it was mentioned in the tree diagram in part 2.

Reply:
Thank you @tayruxin! And yes, the intention is to move DialogBox to part 2 so as to introduce the elements and stucture. The adjustment done to part 2 can be seen in this commit - most of it is very similar to part 3 iterations 1 and 2.